src/ontology/uberon-edit.obo

```diff
@@ -224008,6 +224008,53 @@ relationship: part_of UBERON:0000101 ! lobe of lung
 relationship: proximally_connected_to UBERON:0002184 ! segmental bronchus
 property_value: dcterms-date "2023-03-14T14:54:15Z" xsd:dateTime
 
+[Term]
+id: UBERON:8600010
+name: bronchial submucosal gland ciliated duct
+def: "A ciliated duct in a bronchial submucosal gland." [PMID:30864819]
+intersection_of: UBERON:8600014 ! submucosal gland ciliated duct
+intersection_of: part_of UBERON:0002185 ! bronchus
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489
+relationship: dc-contributor https://orcid.org/0000-0002-9185-3994
+property_value: dcterms-date "2023-05-02T08:07:56Z" xsd:dateTime
+
+[Term]
+id: UBERON:8600011
+name: tracheal submucosal gland ciliated duct
+def: "A ciliated duct in a tracheal submucosal gland." [PMID:30864819]
+intersection_of: UBERON:8600014 ! submucosal gland ciliated duct
+intersection_of: part_of UBERON:0003126 ! trachea
```

Comment:
Consider breaking this up into subclasses:
- 'submucosal gland ciliated duct'
- 'part of' some ‘submucosa of trachea’